Source/Scene/Cesium3DTileFeature.js

```diff
@@ -0,0 +1,247 @@
+/*global define*/
+define([
+        '../Core/Color',
+        '../Core/defined',
+        '../Core/defineProperties'
+    ], function(
+        Color,
+        defined,
+        defineProperties) {
+    'use strict';
+
+    /**
+     * Provides access to a feature's properties stored in the tile's batch table, as well
+     * as the ability to show/hide a feature and change its highlight color via
+     * {@link Cesium3DTileFeature#show} and {@link Cesium3DTileFeature#color}, respectively.
+     * <p>
+     * Modifications to a <code>Cesium3DTileFeature</code> object have the lifetime of the tile's
+     * content.  If the tile's content is unloaded, e.g., due to it going out of view and needing
+     * to free space in the cache for visible tiles, listen to the {@link Cesium3DTileset#tileUnload} event to save any
+     * modifications. Also listen to the {@link Cesium3DTileset#tileVisible} event to reapply any modifications.
+     * </p>
+     * <p>
+     * Do not construct this directly.  Access it through {@link Cesium3DTileContent#getFeature}
+     * or picking using {@link Scene#pick} and {@link Scene#pickPosition}.
+     * </p>
+     *
+     * @alias Cesium3DTileFeature
+     * @constructor
+     *
+     * @example
```

Comment:
We'll be deciding this soon as part of @rahwang's outreach work, but I think it would be better to link to a Sandcastle example heree specifically to show off `Cesium3DTileFeature` instead of having an inline code example.